php/php.editor/src/org/netbeans/modules/php/editor/typinghooks/PhpTypedBreakInterceptor.java

```diff
@@ -159,9 +159,14 @@ public void insert(MutableContext context) throws BadLocationException {
                         PHPTokenId.PHP_LINE_COMMENT));
                 if (helpToken.id() == PHPTokenId.PHP_TOKEN
                         && (helpToken.text().charAt(0) == ',' || helpToken.text().charAt(0) == '(' || helpToken.text().charAt(0) == '[') && ts.movePrevious()) {
-                    // only in array declaration we will add new line
+                    // e.g.
+                    // $array = [^];
+                    // $array = array(^);
+                    // function something(^){};
                     if (helpToken.text().charAt(0) == '[') {
                         sb.append("\n"); // NOI18N
+                    } else if (helpToken.text().charAt(0) == '(') {
+                        sb.append("\n"); // NOI18N
```

Comment:
I think the users prefer to add a new line if there is no parameter. So, I've changed the behavior.
```php
// there is no parameter
// ^: caret position
class Example {
    public function __contruct(^){}
}

// before: don't add the new line
class Example {
    public function __contruct(
            ^){}
}

// after: add the new line
class Example {
    public function __contruct(
            ^
    ){}
}
```

Reply:
This is handy, because commonly used formatting of multiline arguments is 

```php
function (
    $param1,
    $param2,
): void {
    // function body
}
```